bisqapps/shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/uicases/startup/SplashPresenter.kt

```diff
@@ -30,18 +30,11 @@ open class SplashPresenter(
     }
 
     private fun navigateToNextScreen() {
-        // TODO: Conditional nav
+        // TODO: Conditional nav - Will implement once we got persistant storage from nish to save flags
```

Comment:
nish is having some issues with his computer, I might be doing jumping on it this week :+1: